### DISCUSSION TITLE
0.14 upgrade from 0.13, ./run.user: line 84: tempio: command not found error

### DISCUSSION THREAD

**Discussion by gadgetbazza** [blakeblackshear/frigate#14143] [Ask A Question] [answered]
Hi,

I updated to 0.14 as per the title today and now the docker container will not start properly.

I've attached the logs, it appears to be erroring after trying to start NGINX.  Any clues as to what I need to do to fix please?

Thanks
Barry
[_frigate_logs-2.txt](https://github.com/user-attachments/files/17247640/_frigate_logs-2.txt)

**Comment by NickM-27** [comment, marked as answer]
This is covered in the FAQ (and release notes), question 5 https://github.com/blakeblackshear/frigate/discussions/12724

**Reply by gadgetbazza** [reply]
Thank you, I read the installation instructions hoping there was something in there, but didn't come across this FAQ, thanks for the quick response.
